server/apps/main/templates/main/about_us.html

```diff
@@ -2,12 +2,13 @@
 {% load static %}
 {% block title %}AutSPACEs - About Us {% endblock %}
 
-{% block body_attributes%} data-spy="scroll" data-target="#content-bar" {% endblock %}
+{% block body_attributes%} {% endblock %}
 
 {% block content %}
 
 <script src="{% static 'js/menu-alignment.js' %}"></script>
 
+
```

Comment:
There's a small (0.5 rem) gap that's appearing between the heading menu item and the subsequent items; apparently added by the `<h5>`.

It looks okay actually, but I think is probably unintentional. Something like this could be used to remove it (added to `main.css`).
```
.list-group-item {
  margin-bottom: 0;
}
```

Reply:
Great catch, I hadn't noticed. I've replaced the h5 with a more default declaration for the list view and made that bold instead, as I think that works well :)